shell/platform/android/io/flutter/plugin/platform/PlatformView.java

```diff
@@ -16,13 +17,39 @@
      */
     View getView();
 
+    /**
+     * An Android {@link View} responsible for rendering a Flutter UI is now
```

Comment:
subjective/optional nit: Passive voice, could be reworded to "Called by this Flutter engine when a view responsible for rendering a Flutter UI is..."

https://developers.google.com/style/voice

Reply:
Thanks. Done.